aptos-move/framework/src/natives/aggregator_natives/context.rs

```diff
@@ -145,50 +155,110 @@ impl AggregatorChangeSet {
-        fn resolve_table_entry(
-            &self,
-            _handle: &TableHandle,
-            _key: &[u8],
-        ) -> Result<Option<Vec<u8>>, anyhow::Error> {
-            Ok(None)
-        }
+    use aptos_aggregator::{
+        aggregator_extension::aggregator_id_for_test, delta_change_set::serialize,
+    };
+    use aptos_language_e2e_tests::data_store::FakeDataStore;
+    use aptos_types::state_store::state_key::StateKey;
+    use claims::{assert_matches, assert_ok};
+
+    fn get_test_resolver() -> FakeDataStore {
+        #[allow(clippy::redundant_closure)]
+        let mut state_view = FakeDataStore::default();
+
+        let AggregatorID {
+            handle: handle_500,
+            key: key_500,
+        } = aggregator_id_for_test(500);
+        let AggregatorID {
+            handle: handle_600,
+            key: key_600,
+        } = aggregator_id_for_test(600);
+        let AggregatorID {
+            handle: handle_700,
+            key: key_700,
+        } = aggregator_id_for_test(700);
+
+        state_view.set_legacy(
+            StateKey::table_item(handle_500.into(), key_500.0.to_vec()),
+            serialize(&150),
+        );
+        state_view.set_legacy(
+            StateKey::table_item(handle_600.into(), key_600.0.to_vec()),
+            serialize(&100),
+        );
+        state_view.set_legacy(
+            StateKey::table_item(handle_700.into(), key_700.0.to_vec()),
+            serialize(&200),
+        );
+        state_view
     }
 
     // All aggregators are initialized deterministically based on their ID,
     // with the following spec.
     //
-    //     +-------+---------------+----------+-----+---------+
-    //     |  key  | storage value |  create  | get | remove  |
-    //     +-------+---------------+----------+-----+---------+
-    //     |  100  |               |   yes    | yes |   yes   |
-    //     |  200  |               |   yes    | yes |         |
-    //     |  300  |               |   yes    |     |   yes   |
-    //     |  400  |               |   yes    |     |         |
-    //     |  500  |               |          | yes |   yes   |
-    //     |  600  |               |          | yes |         |
-    //     |  700  |               |          | yes |         |
-    //     |  800  |               |          |     |   yes   |
-    //     +-------+---------------+----------+-----+---------+
-    fn test_set_up(context: &NativeAggregatorContext) {
+    //     +-------+---------------+-----------+-----+---------+
+    //     |  key  | storage value |  create   | get | remove  |
+    //     +-------+---------------+-----------+-----+---------+
+    //     |  100  |               |   yes     | yes |   yes   |
+    //     |  200  |               |   yes     | yes |         |
+    //     |  300  |               |   yes     |     |   yes   |
+    //     |  400  |               |   yes     |     |         |
+    //     |  500  |               |   Yes     | yes |   yes   |
```

Comment:
haha, sorry, when I see I can't unsee, let's make lower-case Y here or upper-case elsewhere to be consistent :-)